sswlib/src/main/java/components/Mech.java

```diff
@@ -2778,7 +2778,15 @@ public double GetDefensiveFactor() {
         }
 
         MechModifier m = GetTotalModifiers( true, true );
-        retval += m.DefensiveBonus();
+        // Apply void sig errata
+        if (HasVoidSig) {
+            if (retval < 1.3) {
+                retval += 0.3;

```

Comment:
At some point, possibly during refactoring stages, it might be good to move these values into someplace a little easier to maintain. I'm just thinking that with every errata, maintainers will need to do a find and replace. I'm thinking it would be good for each equipment / weapon / etc to be able to do it's own calculation. Then maintainers can go right to the equipment when there is errata or rule changes. What do you think?

Reply:
Definitely agree it should be refactored. I was planning to look at that after the 0.7 release, but in the meantime while we address bugfixes I've just been sticking to the structure the previous maintainers used (see [this commit](https://github.com/Solaris-Skunk-Werks/sswlib/commit/ff17c6f0eeaf5de4487305e5a8028d930e871ca7) where they added the stealth armor handler)

Reply:
Totally makes sense! Just had a thought I wanted to share 🙂